pkg/server/api/bundle/v1/service.go

```diff
@@ -436,9 +436,9 @@ func (s *Service) updateFederatedBundle(ctx context.Context, b *types.Bundle, in
 	switch status.Code(err) {
 	case codes.OK:
 	case codes.NotFound:
-		log.WithError(err).Error("Bundle not found")
+		log.Error("Bundle not found")
```

Comment:
Should we remove `err` when it is a `codes.AlreadyExists` as well?

Reply:
Yeah, good catch.